atom/browser/atom_browser_client.cc

```diff
@@ -512,6 +514,16 @@ void AtomBrowserClient::AppendExtraCommandLineSwitches(
   }
 }
 
+void AtomBrowserClient::AdjustUtilityServiceProcessCommandLine(
+    const service_manager::Identity& identity,
+    base::CommandLine* command_line) {
+#if defined(OS_MACOSX)
+  if (identity.name() == video_capture::mojom::kServiceName ||
```

Comment:
Perhaps store identity.name() in a var to avoid calling it twice.

Reply:
@shahzadlone this is an automated backport of a PR that was previously approved; if we were going to make changes it would be on the original PR prior to its being merged.